src/components/Repos/Repo.js

```diff
@@ -1,45 +1,72 @@
 import { useState, useEffect } from 'react'
-import { useParams } from "react-router-dom"
+import { useParams, Link } from "react-router-dom"
 
 function Repo () {
-  const [repo, setRepo] = useState({})
+  const [repo, setRepo] = useState({notes: []})
```

Comment:
This kind of approach didn't even occur to me, using filtering on the fetch requests to build an array of all notes for the current repo. My method was to have one state for Repo, with the current Repo name stored in, and one state of Notes which contained all notes in the json, and then filter the notes array inside my map function when displaying them.

Your method is much more efficient!